File: app/src/main/java/me/moodi1999/example/ArgumentBuilderApplication.java
package me.moodi1999.example;

import android.app.Application;
import android.content.Context;
import androidx.multidex.MultiDex;

public class ArgumentBuilderApplication extends Application {

    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
        MultiDex.install(this);
    }
}

File: app/src/main/java/me/moodi1999/example/ArgumentBuilderApplication.kt
package me.moodi1999.example

import android.app.Application
import android.content.Context
import androidx.multidex.MultiDex

class ArgumentBuilderApplication : Application() {
    override fun attachBaseContext(base: Context) {
        super.attachBaseContext(base)
        MultiDex.install(this)
    }
}